install_radar_deps.py

```diff
@@ -1,4 +1,4 @@
-#!/bin/python3
+#!/usr/bin/env/python3
```

Comment:
env is a program and you give it an argument like so
```suggestion
#!/usr/bin/env python3
```